### PR TITLE
#779 Show the download icon of bintray repo

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #779)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
Only add badge
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Easy to deploy the streaming application
 
 [![Build Status](https://builds.is-land.com.tw/buildStatus/icon?job=PostCommit-OHARA)](https://builds.is-land.com.tw/job/PostCommit-OHARA/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Download](https://api.bintray.com/packages/oharastream/ohara/ohara-kafka/images/download.svg)](https://bintray.com/oharastream/ohara)
```
Comment (vitojeng) [inline README.md]: `Download` 是不是改成 `Bintray` or `Maven Repo` 比較好?
`Download` 我比較傾向是指 Github release page 裡的 Assets

Comment (chia7712) [reply README.md]: +1

Comment (saivirtue) [reply README.md]: `Maven Repo` is fine :)

Comment (mathsigit) [reply README.md]: How about using the url to get `bintray`
https://img.shields.io/bintray/v/oharastream/ohara/ohara-client.svg
and it looks like bintray

Comment (chia7712) [reply README.md]: @mathsigit great suggestion!

Comment (saivirtue) [reply README.md]: OK, edit right away

**Comment by saivirtue** [comment]
Does this look fine? I will merge this fancy badge soon after

**Comment by vitojeng** [comment]
+1

**Pull request merged**